libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -925,6 +926,26 @@ bool AP_Arming::camera_checks(bool display_failure)
     return true;
 }
 
+bool AP_Arming::osd_checks(bool display_failure) const
+{
+    if ((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_CAMERA)) {
```

Comment:
Is `CAMERA` a good fit here?

There's nothing better?

Reply:
I think CAMERA was your suggestion on a previous review 😄 

It depends how granular we want to get. I think camera is ok unless you want a specific OSD one.